doctrine/event_listeners_subscribers.rst

```diff
@@ -169,8 +169,8 @@ interface and have an event method for each event it subscribes to::
     namespace AppBundle\EventListener;
 
     use Doctrine\Common\EventSubscriber;
-    use Doctrine\ORM\Event\LifecycleEventArgs;
-    // for Doctrine 2.4: Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+    // for Doctrine < 2.4: use Doctrine\ORM\Event\LifecycleEventArgs;
```

Comment:
The **use Doctrine\ORM\Event\LifecycleEventArgs;** need to be use for doctrine before doctrine 2.4.
For doctrine >= 2.4 we need to **use Doctrine\Common\Persistence\Event\LifecycleEventArgs;**